repondeur/tests/models/test_lecture.py

```diff
@@ -3,10 +3,67 @@
 import pytest
 import transaction
 
+from zam_repondeur.models.chambre import Chambre
+
 # We need data about dossiers, texts and groups
 pytestmark = pytest.mark.usefixtures("data_repository")
 
 
+@pytest.fixture
+def lecture_ccfp(db):
```

Comment:
It's not clear to me if it is pertinent to reuse fixtures from https://github.com/betagouv/zam/pull/1087/files#diff-5c85a9c69aaac9918b965ba3589eecb7R23-R66